dll/cpl/desk/lang/en-US.rc

```diff
@@ -136,6 +136,23 @@ BEGIN
     PUSHBUTTON "Ad&vanced...", IDC_SETTINGS_ADVANCED, 170, 174, 70, 15
 END
 
+IDD_ADVANCED_GENERAL DIALOGEX 0, 0, 253, 204
+STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
+CAPTION "General"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX "Display:", -1, 7, 6, 237, 105, WS_GROUP
+    LTEXT  "If the screen resolution makes items too small to view comfortably, you can increase the ppp value to compensate. To change the font size only, click ""Cancel"" and go to the ""Appearance"" tab.", -1, 14, 20, 220, 45
+    LTEXT "Font Size:", -1, 14, 65, 222, 9
+    COMBOBOX IDC_FONTSIZE_COMBO, 14, 75, 223, 80, CBS_DROPDOWNLIST | WS_VSCROLL | WS_TABSTOP
+    LTEXT "", IDC_FONTSIZE_CUSTOM, 14, 91, 223, 12
+    GROUPBOX "Compatibility", -1, 7, 115, 237, 100
+    LTEXT "Some programs may not function properly if you do not restart your computer after changing the display settings.\n\nAfter changing the display settings:", -1, 12, 130, 228, 34
+    AUTORADIOBUTTON "& Restart the computer before applying the new settings", IDC_RESTART_RB, 14, 165, 228, 10
+    AUTORADIOBUTTON "& Apply new settings without restarting", IDC_WITHOUTREBOOT_RB, 14, 180, 228, 10
```

Comment:
What are these `&`s?
Are they for mnemonics? If so, there can't be a space after the `&`, and you also can't use the letter `A` twice (this line and the one below).

Reply:
It was my mistake (the space)